randovania/exporter/game_exporter.py

```diff
@@ -102,8 +102,12 @@ def export_game(
                     input_hash = export_params.calculate_input_hash()
                     incorrect_hashes = self.get_unknown_input_hashes(input_hash)
                     if incorrect_hashes:
-                        msg = "An error occurred while exporting.\nThe following files have invalid hashes:\n"
-                        msg += "\n".join(f"{key} => {value}" for key, value in incorrect_hashes.items())
+                        msg = "An error occurred while exporting.\n\nYour following input is invalid:\n"
```

Comment:
another slight nit: the title says "error during exporting" while this says "error [occured] while exporting"
would be nice to have them consistent.